### ISSUE TITLE
Component: Navbar Component (including sign in and create account buttons) (This task is part of User Story #41)

### ISSUE THREAD

**Issue by JonathanSussman1** [agile-dev-assignments/project-setup-team-ntobo-global#12]
See #41

**Comment by Dendrobyte** [comment]
Note to self: Make a new task to link Olivia's hamburger menu when both are completed

**Comment by Dendrobyte** [comment]
Right now just functional. Once hamburger component is done we can go ahead and use that, or I can just implement it via drop down. Can make signin/signout buttons look a little better, but if I have time toward the end of the sprint.
